membership-attribute-service/app/services/SupporterRatePlanToAttributesMapper.scala

```diff
@@ -160,6 +160,10 @@ object SupporterRatePlanToAttributesMapper {
       "2c92a0fd57d0a9870157d7412f19424f",
       "2c92a0fe57d0a0c40157d74241005544",
       "2c92a0ff58bdf4eb0158f307ed0e02be",
+      "2c92a0fd79ac64b00179ae3f9d474960",
```

Comment:
Are these old GW plans?

Reply:
I copied all (missing ids) from support-frontend.

Reply:
They're "GW Oct 18 *" plans, whatever that means.

Reply:
That's 2018 right?